script/tool/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
IIRC we'll need to publish this fix so it can be used from flutter/packages, right?

```suggestion
## 0.8.5
```

Reply:
I did this because I have two different PRs in flight that need to be published for flutter/packages, and my plan is to just land this one first and bundle them into one version change to make conflict resolution slightly easier.

Reply:
Perfect!